.travis.yml

```diff
@@ -62,6 +62,10 @@ matrix:
     - jdk: "oraclejdk7"
       env: TEST_SELENIUM="true" SCALA_VER="2.10" SPARK_VER="1.6.1" HADOOP_VER="2.3" PROFILE="-Pspark-1.6 -Phadoop-2.3 -Ppyspark -Pexamples" BUILD_FLAG="package -DskipTests -DskipRat" TEST_FLAG="verify -DskipRat" TEST_PROJECTS="-pl zeppelin-interpreter,zeppelin-zengine,zeppelin-server,zeppelin-display,spark-dependencies,spark -Dtest=org.apache.zeppelin.AbstractFunctionalSuite -DfailIfNoTests=false"
 
+    # Test livy with spark 1.5.2 and hadoop 2.6
+    - jdk: "oraclejdk7"
+      env: SCALA_VER="2.10" SPARK_VER="1.5.2" HADOOP_VER="2.6" LIVY_VER="0.2.0" PROFILE="-Pspark-1.5 -Phadoop-2.6" BUILD_FLAG="package -DskipTests -DskipRat" TEST_FLAG="verify -DskipRat" TEST_PROJECTS="-pl zeppelin-interpreter,livy -DfailIfNoTests=false"
```

Comment:
livy integration test doesn't work with hadoop-2.3. If I merge livy integration with existing test profile, I have to change hadoop version from 2.3 to 2.6, not sure whether it is acceptable. @Leemoonsoo